SuiteCRM/SuiteCRM
# Print View on Browser shows Links on Listview

## Description




When on Module Listview you can right-click and print, when the print screen appears the list view was not displaying like the Listview normally, with Suite Links populating values. Now using content: none !important; on a[href]:after takes away the links and just displays the record data.

## Motivation and Context

Client Raised this issue 

## How To Test This


1. Navigate to a Listview, and rightclick and select print.
2. See that there is a copy of the list view and no URL's are shown.
You may need to change the size of the print page to show all data on the list view. For Chrome I changed it to "Tabloid" to get all the Listview Data in view.

## Types of changes


### Final checklist
